bedrock/firefox/templates/firefox/lockwise/lockwise.html

```diff
@@ -0,0 +1,83 @@
+{% add_lang_files "firefox/products/lockwise" %}
+
+{% from "macros.html" import google_play_button with context %}
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{_('Firefox Lockwise — password manager —  take your passwords everywhere')}}{% endblock %}
+
+{% block page_desc %}{{ _('Firefox Lockwise lets you securely access the passwords you’ve saved in Firefox from anywhere— even outside of the browser. Features 256-bit encryption and Face/Touch ID.') }}{% endblock %}
```

Comment:
@Errolyn you need to add an extra space before the M dash, the changes you made in the file for l10n. Otherwise the string won't show up. It needs to match. 

`...from anywhere— even outside ...` to .`... from anywhere — even outside ...`

Reply:
thanks for the heads up!